src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -366,7 +366,6 @@ private function addHttpCacheSection(ArrayNodeDefinition $rootNode): void
                         ->end()
                         ->booleanNode('public')->defaultNull()->info('To make all responses public by default.')->end()
                         ->arrayNode('invalidation')
-                            ->setDeprecated('The use of the `http_cache.invalidation` has been deprecated in 2.6 and will be removed in 3.0. Use `defaults.cache_headers.invalidation` instead.')
```

Comment:
Why?

Reply:
Because AFAIU it’s not an attribute, so setting it in defaults is useless.